src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/StageAppYamlTask.java

```diff
@@ -19,12 +19,17 @@
 
 import com.google.cloud.tools.appengine.AppEngineException;
 import com.google.cloud.tools.appengine.operations.AppYamlProjectStaging;
+import javax.inject.Inject;
 import org.gradle.api.DefaultTask;
+import org.gradle.api.internal.file.FileOperations;
 import org.gradle.api.tasks.Nested;
 import org.gradle.api.tasks.TaskAction;
 
 /** Stage App Engine app.yaml based applications for deployment. */
-public class StageAppYamlTask extends DefaultTask {
+public abstract class StageAppYamlTask extends DefaultTask {
+
+  @Inject
+  abstract public FileOperations getFileOperations();
```

Comment:
`FileSystemOperations` requires Gradle 6.0+
Recently there was a commit comparing the version with 5.5 (see https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/451 ), so I'm not sure if bumping the minimal version to Gradle 6.0 is generally expected from an innocent change like this

```suggestion
  abstract public FileSystemOperations getFileSystemOperations();
```

Reply:
Don't take my usage of 5.1 as a guide :) I used the lowest possible point where the new API is available, this makes the condition disappear faster than if it's at the point of deprecation during maintenance.

A more relevant discussion might be: https://github.com/GoogleCloudPlatform/app-gradle-plugin/issues/450